src/main/java/rsb/internal/globval/GlobalConfiguration.java

```diff
@@ -149,7 +160,7 @@ public static String getUnixHome() {
 		}
 	}
 
-	public static final String NAME = "GService";
+	public static final String NAME = "OsrsBot";
```

Comment:
Should never have been called GService (BREAKING CHANGE) - People will have to move their stuff

Reply:
Better now than later tho

Reply:
Hard agree.